src/makeCalculator.js

```diff
@@ -5,6 +5,35 @@
  */
 function makeCalculator() {
```

Comment:
The function name `makeCalculator` is appropriate as per the naming conventions in the checklist.